benchmarks/crmarena/executor.py

```
@@ -813,338 +841,122 @@ def _extract_structured_text(self, response_text: str) -> Any:
+                    # Determine parameter type from annotation
+                    param_type = "string"  # default
+                    param_description = f"Parameter {param_name}"
+                    
+                    # Get type from annotation if available
+                    if param.annotation != inspect.Parameter.empty:
+                        annotation = param.annotation
+                        if hasattr(annotation, '__origin__'):
+                            if annotation.__origin__ is list:
+                                param_type = "array"
+                                if hasattr(annotation, '__args__') and annotation.__args__:
+                                    if annotation.__args__[0] == dict:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)